detection/include/detection/detect_platform.hpp

```diff
@@ -3,21 +3,21 @@
 #include <cv_bridge/cv_bridge.h>
 #include <geometry_msgs/Point.h>
 #include <image_transport/image_transport.h>
-#include <nav_msgs/Odometry.h>
 #include <opencv2/opencv.hpp>
 #include <ros/ros.h>
 #include <sensor_msgs/Image.h>
 #include <sensor_msgs/image_encodings.h>
+#include <std_msgs/Bool.h>
 #include <xmlrpcpp/XmlRpc.h>
 
-namespace ariitk::auto_landing {
+namespace ariitk::detection {
 class PlatformDetect {
```

Comment:
space here

Reply:
Where? Do you mean a newline after the namespace?

Reply:
sorry, yes i meant line

Reply:
Where?